app/views/linkers/index.html.erb

```diff
@@ -8,72 +8,26 @@
   </div>
   <div class="form-group">
     <%= label_tag %>
-    <%= text_field_tag 'search_term', "", {:placeholder => 'Search term here', :class => 'form-control'} %>
+    <%= text_field_tag 'search-term', "", {:placeholder => 'Search term here', :class => 'form-control'} %>
   </div>
   <%= submit_tag 'Process URLs', class: 'btn btn-default' %>
-  <%= link_to 'Test', 'javascript:void(0)', :onclick => 'readCSVFile()', class: 'btn btn-default' %>
+  <button class='btn btn-default' id='test-button'>Test</button>
```

Comment:
It's not best practice to use `onclick` directly in the HTML, but rather to attach events to a click listener. That way, the HTML and JS aren't too tied together.